src/VolleyManagement.Backend/VolleyManagement.UI/Global.asax.cs

```diff
@@ -29,7 +31,9 @@ public void FixEfProviderServicesProblem()
             ////Make sure the provider assembly is available to the running application.
             ////See http://go.microsoft.com/fwlink/?LinkId=260882 for more information.
 
+#pragma warning disable S1481 // Unused local variables should be removed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)